go.mod

```diff
@@ -213,6 +213,8 @@ require (
 	modernc.org/sqlite v1.24.0
 )
 
+require github.com/caio/go-tdigest/v4 v4.0.1 // indirect
```

Comment:
Can you remove this extra require section and include this with the other indrection deps please?